### ISSUE TITLE
TypeError: callback is not a function with empty data

### ISSUE THREAD

**Issue by beyondcompute** [olalonde/follow-redirects#91]
Hello and thank you for your work!

When I do a request with empty buffer (using axios), I get an error (`follow-redirects/index.js`).

Code to reproduce:
```js
const axios = require('axios');

const form = new FormData();
const fileBuffer = Buffer.from([]);
form.append('file', fileBuffer, 'example.txt');

try {
  await axios.post('https://example', form);
} catch (err) {
  console.error(err);
}
```
Same code on Runkit: https://runkit.com/embed/hereta5r67jc

<details>
<summary>Example stack trace:</summary>

```
TypeError: callback is not a function
      at RedirectableRequest.write (node_modules/follow-redirects/index.js:70:5)
      at FormData.ondata (internal/streams/legacy.js:16:26)
      at FormData.CombinedStream.write (node_modules/combined-stream/lib/combined_stream.js:118:8)
      at FormData.CombinedStream._pipeNext (node_modules/combined-stream/lib/combined_stream.js:106:8)
      at FormData.CombinedStream._getNext (node_modules/combined-stream/lib/combined_stream.js:79:10)
      at FormData.CombinedStream._pipeNext (node_modules/combined-stream/lib/combined_stream.js:107:8)
      at FormData.CombinedStream._getNext (node_modules/combined-stream/lib/combined_stream.js:79:10)
      at FormData.CombinedStream._pipeNext (node_modules/combined-stream/lib/combined_stream.js:107:8)
      at FormData.<anonymous> (node_modules/combined-stream/lib/combined_stream.js:91:10)
      at FormData.<anonymous> (node_modules/form-data/lib/form_data.js:285:5)
      at FormData.CombinedStream._getNext (node_modules/combined-stream/lib/combined_stream.js:84:3)
      at FormData.CombinedStream._pipeNext (node_modules/combined-stream/lib/combined_stream.js:107:8)
      at FormData.CombinedStream._getNext (node_modules/combined-stream/lib/combined_stream.js:79:10)
      at FormData.CombinedStream._pipeNext (node_modules/combined-stream/lib/combined_stream.js:107:8)
      at FormData.CombinedStream._getNext (node_modules/combined-stream/lib/combined_stream.js:79:10)
      at FormData.CombinedStream.resume (node_modules/combined-stream/lib/combined_stream.js:134:10)
      at FormData.CombinedStream.pipe (node_modules/combined-stream/lib/combined_stream.js:64:8)
      at dispatchHttpRequest (node_modules/axios/lib/adapters/http.js:269:10)
      at new Promise (<anonymous>)
      at httpAdapter (node_modules/axios/lib/adapters/http.js:20:10)
      at dispatchRequest (node_modules/axios/lib/core/dispatchRequest.js:59:10)
      at <anonymous>
```
</details>

I tried with  _axios_ `0.18.0` and different versions of _follow-redirects_ and the error only appears in `1.5.2`. It does not reproduce in `1.5.1`.

Now, I am not sure, if it’s an axios’ bug or does it belong here. I hope, you could help.

Thank you and have a good week!

**Comment by RubenVerborgh** [comment]
Right, this was indeed introduced by #90; we shouldn't assume a callback has been passed.

**Comment by RubenVerborgh** [comment]
Released as v1.5.3.

**Comment by beyondcompute** [comment]
Thank you for the quick fix and fast release!
